spec/gameobj-data/boxes_and_picking_spec.rb

```diff
@@ -59,12 +88,17 @@
 
     end
 
-    boon_boxes = boon_box_nouns.product(boon_box_prefixes, boon_box_after_names)
+    boon_boxes = boon_box_nouns.product(boon_box_materials, boon_box_descriptions)
 
-    boon_boxes.each do |noun, prefix, after_name|
-      it "recognizes #{prefix} #{noun} #{after_name} as a box" do
-        box = GameObjFactory.item_from_name("#{prefix} #{noun}", noun, after_name)
+    boon_boxes.each do |noun, material, desc|
+      full_boon_box_description = "#{desc} #{material} #{noun}"
+      it "recognizes #{full_boon_box_description} as a box" do
+        box = GameObjFactory.item_from_name(full_boon_box_description)
         expect(box.type).to eq "box"
+      # Removed because long description should not need to be counted - ???
```

Comment:
We do not currently have anything in gameobj-data.xml that looks at `after_name`, but my intention with the tests is to create items that are as close as possible to what we will encounter in the game. Since you went through the work to collect the full descriptions, I think it makes sense to use the full item name in the tests.